.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+- package-ecosystem: cargo
+  directory: "/"
+  schedule:
+    interval: weekly
+  open-pull-requests-limit: 10
```

Comment:
This can be adjusted down if desired.

Reply:
I think 5 is good. Keep things on major versions only and don't automatically update the cargo.lock, there's no substitute for knowing what deps tree everyone is using in production. If you don't keep track you'll at some point regret it.